### ISSUE TITLE
tap_action not working

### ISSUE THREAD

**Issue by apbarratt** [MindFreeze/ha-sankey-chart#115]
**Checklist:**

- [ X ] I updated to the latest version available
- [ X ] I cleared the cache of my browser

**Release with the issue:**
[v1.13.0](https://github.com/MindFreeze/ha-sankey-chart/releases/tag/v1.13.0)

**Last working release (if known):**
Feature was introduced in last update

**Browser and Operating System:**

Safari on Intel Mac running latest OS updates: 13.3.1 (a) (22E772610a)

**Description of problem:**

Hi guys, the new tap_action per entity feature does not appear to work and I think I've followed the documentation correctly.  Each entity it is applied to continues to use the default action of more info.

In the following code, I have attempted to add tap_actions in four locations.  The first three are meant to navigate to a different dashboard when clicked.  The fourth was a test to check if the problem existed on other action types and is just a toggle, matching the example given in the documentation.  All four entities, when clicked, continue to just show the more info modal.  Including the remaining_parent_state one, which continues to show non-useful information as before.

```yaml
title: Usage right now
type: custom:sankey-chart
height: 200
show_names: true
show_icons: true
min_box_height: 28
min_box_width: 28
wide: false
sections:
  - entities:
    - entity_id: sensor.power_from_grid
      name: Grid
      color: red
      children:
        - sensor.office_meter_phase_a_power
        - sensor.battery_charge
        - house
    - entity_id: sensor.power_pv_array
      name: Solar
      color: gold
      children:
        - sensor.office_meter_phase_a_power
        - sensor.battery_charge
        - sensor.power_to_grid
        - house
      tap_action:
        action: navigate
        navigation_path: /skua-drive/solar
    - entity_id: sensor.battery_discharge
      name: Battery usage
      color: orange
      children:
        - sensor.office_meter_phase_a_power
        - house
      tap_action:
        action: navigate
        navigation_path: /skua-drive/solar
  - entities:
    - entity_id: sensor.office_meter_phase_a_power
      name: Office
      children:
        - sensor.ups_supply_energy_power
        - sensor.desk_energy_power
        - sensor.office_floor_power
        - sensor.office_heater_power
        - unknown_office
    - entity_id: house
      name: House
      icon: mdi:home
      type: remaining_parent_state
      tap_action:
        action: navigate
        navigation_path: /skua-drive/solar
    - entity_id: sensor.battery_charge
      name: Battery charging
      color: gold
    - entity_id: sensor.power_to_grid
      name: Exporting
      color: orange
  - entities:
    - entity_id: sensor.ups_supply_energy_power
      name: UPS
      children:
        - sensor.3d_printer_power_energy_power
    - entity_id: sensor.desk_energy_power
      name: Desk
      tap_action:
        action: toggle
    - entity_id: sensor.office_floor_power
      name: Floor
      color: orange
    - entity_id: sensor.office_heater_power
      name: Heater
      color: red
    - entity_id: unknown_office
      type: remaining_parent_state
      name: Other office
  - entities:
    - entity_id: sensor.3d_printer_power_energy_power
      name: 3D Printer
      color: orange
view_layout:
  position: main
```

**Javascript errors shown in the web inspector (if applicable):**

```
n/a
```

**Additional information:**
I've tried refreshing the page and loading on different devices but the problem persists.

**Comment by apbarratt** [comment]
Hold on, let me figure out what's gone wrong, somehow all the work I've done has reverted to original :/

**Comment by apbarratt** [comment]
Sorry about that, I've edited it with the correct yaml now.  No idea how that happened, but rewritten it all and can confirm tap_action continues to not work.

**Comment by MindFreeze** [comment]
Have you tried other browsers? I don't have apple devices to try Safari and can't reproduce this in Chromium and Firefox.

**Comment by apbarratt** [comment]
Sorry for the delay.  This hasn't worked for 3 days (and I was trying quite regularly) and then this morning when I finally sat down to test it in Chrome for you it was working there.  So I switched back to Safari and it was now working there too.  So I switched over to my iPhone app and it is now working there too.  I am absolutely flummoxed as to what was causing it.
The one clue I can think of is that last night, before bed, I installed a couple of updates to stuff that probably triggered a restart overnight so perhaps some cache busting took place that hadn't taken hold before hand :/
I'll close this issue for now and will come back to it if I manage to reproduce the issue again.
